src/__tests__/App.spec.tsx

```diff
@@ -28,4 +28,36 @@ test("loads and displays header", async () => {
   );
 
   expect(await screen.findByText("Posts")).toBeInTheDocument();
+  expect(await screen.findByText("Create a new post")).toBeInTheDocument();
+  expect(await screen.findByText("Delete post")).toBeInTheDocument();
+  expect(await screen.findByText("Update post")).toBeInTheDocument();
+});
+
+test("loads and displays post", async () => {
+  const mocks: any = [
```

Comment:
Running the tests now renders [this error](https://www.apollographql.com/docs/react/errors#%7B%22version%22%3A%223.8.9%22%2C%22message%22%3A12%2C%22args%22%3A%5B%22user%22%2C%22%7B%5Cn%20%20%5C%22title%5C%22%3A%20%5C%22abcd%5C%22%2C%5Cn%20%20%5C%22body%5C%22%3A%20%5C%22qwerty%5C%22%2C%5Cn%20%20%5C%22id%5C%22%3A%20%5C%22a123%5C%22%5Cn%7D%22%5D%7D). How would you solve it?